scripts/generate-openapi-bindings-bdd.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -e
+
+ROOT_DIR="$(dirname "$0")/.."
+echo $ROOT_DIR
+TARGET="$ROOT_DIR/tests/bdd/openapi"
+SPEC="$ROOT_DIR/control-plane/rest/openapi-specs/v0_api_spec.yaml"
+
+# Regenerate the bindings only if the rest src changed
+check_spec="no"
+
+case "$1" in
+    --changes)
+        check_spec="yes"
+        ;;
+esac
+
+if [[ $check_spec = "yes" ]]; then
+    git diff --cached --exit-code "$SPEC" 1>/dev/null && exit 0
+fi
+
+# Cleanup the existing autogenerated code
+if [ ! -d "$TARGET" ]; then
+  mkdir -p "$TARGET"
+else
+  rm -rf "$TARGET"
+  mkdir -p "$TARGET"
+fi
+
+# Generate a new openapi python client for use by the BDD tests
+openapi-generator-cli generate -i "$SPEC" -g python -o "$TARGET"
+
+# If the openapi bindings were modified then fail the check
```

Comment:
we don't need to add this since we're not committing the files

Reply:
Removed.